.github/workflows/CI.yml

```diff
@@ -147,43 +147,29 @@ jobs:
     name: Linux
     runs-on: ubuntu-20.04
     needs: check_changelog
-    strategy:
-      fail-fast: true  # false to test all, true to fail entire job if any fail
-      matrix:
-        distro: [ debian, ubuntu_18_04, ubuntu_20_04, ubuntu_21_10 ]
-        package: [ -p ]
-        extension: [ deb ]
-        include:  # package these differently
-          - distro: fedora_35
-            package: '-p'
-            extension: rpm
-
     steps:
       - name: Checkout
         uses: actions/checkout@v3
         with:
           submodules: recursive
 
```

Comment:
Need to install build time dependencies. Can likely combine the the appimage build here as well.